site/docs/ch/flink/flink-ddl.md

```diff
@@ -16,6 +16,12 @@ CREATE CATALOG <catalog_name> WITH (
 |metastore.url|(none)|String|是|Arctic Metastore 的 URL，thrift://`<ip>`:`<port>`/`<catalog_name_in_metastore>`|
 |default-database<img width=100/>|default|String|否|默认使用的数据库|
 |property-version|1|Integer|否|Catalog properties 版本，此选项是为了将来的向后兼容性|
+| properties.auth.load-from-ams | True | BOOLEAN | 否 |是否从 AMS 加载安全校验的配置。 true:从AMS加载；false:不使用AMS的配置。注意：不管该参数是否配置，只要用户配置了下述 auth.*** 相关参数，就会使用该配置进行访问|
+| properties.auth.type | NULL | String | 否 | 表安全校验类型，有效值：simple、kerberos 或不配置。默认不配置，无需权限检验。simple：使用 hadoop 用户名，搭配参数'properties.auth.simple.hadoop_username'使用； kerberos：配置 kerberos 权限校验，搭配参数'properties.auth.kerberos.principal','properties.auth.kerberos.keytab','properties.auth.kerberos.krb'使用 |
```

Comment:
Please use the default `(none)` to replace the `NULL`.
Aligned with others' configurations.